easybuild/easyconfigs/s/singularity/singularity-3.8.0-daint.eb

```diff
@@ -0,0 +1,39 @@
+# @author: teojgo gppezzi
+easyblock = 'Bundle'
+
+name = 'singularity'
+version = '3.8.0'
+versionsuffix = '-daint'
+
+homepage = 'https://www.sylabs.io/'
+description = """Singularity is a portable application stack packaging and runtime utility."""
+
+toolchain = SYSTEM
+
+modtclfooter = """
+# Swap the programming environment
+module unload PrgEnv-cray
+module unload PrgEnv-intel
+module unload PrgEnv-pgi
```

Comment:
@victorusu do you think that PrgEnv-nvidia should also be added here?

Reply:
Yes